### ISSUE TITLE
Operational Issues

### ISSUE THREAD

**Issue by ZorryTheKid** [RevertIT/mybb-rt_chatgpt#18]
.Regarding the settings as shown in the attached image, the moderation effect works on entire forums, meaning that when a new topic is opened containing offensive words (I don't know if in the title or in the post content), it gets placed in moderation and not shown. With these settings, posts within already active and present Topics are not moderated. If a user writes offensive words, in response to others, it won't get moderated.
![Screenshot 2024-08-22 at 12-26-27 Impostazioni della Board - RT ChatGPT Assistant](https://github.com/user-attachments/assets/4871c63f-a31a-48dd-a81b-bd682d953183)

**Comment by RevertIT** [comment]
Yeah, the post moderation was not added, only the thread moderation.
